src/backend/distributed/executor/local_executor.c

```diff
@@ -207,19 +207,6 @@ ExecuteLocalTaskListExtended(List *taskList,
 	uint64 totalRowsProcessed = 0;
 	ParamListInfo paramListInfo = copyParamList(orig_paramListInfo);
 
-	/*
-	 * Even if we are executing local tasks, we still enable
-	 * coordinated transaction. This is because
-	 *  (a) we might be in a transaction, and the next commands may
-	 *      require coordinated transaction
-	 *  (b) we might be executing some tasks locally and the others
-	 *      via remote execution
```

Comment:
Why does this part of the comment not apply anymore?

Reply:
> 	 *  (a) we might be in a transaction, and the next commands may
	 *      require coordinated transaction

Next commands is already (and should be) covered via [Activate2PCIfModifyingTransactionExpandsToNewNode](https://github.com/citusdata/citus/blob/0944c631f1eaa5b51478631ab40c1b4ad57d2cd2/src/backend/distributed/executor/adaptive_executor.c#L3832)

> 	 *  (b) we might be executing some tasks locally and the others
	 *      via remote execution
In the old days, we first executed local commands and then remote ones.  But, as of https://github.com/citusdata/citus/pull/4301, we do the order the other way. First remote, then local. The remote tasks (e.g., multi-shard commands) already set 2PC if needed.